src/tools/pmempool/dump.c

```diff
@@ -304,10 +296,20 @@ pmempool_dump_func(char *appname, int argc, char *argv[])
 			pd.hex = 0;
 			break;
 		case 'r':
-			pd.str_ranges = optarg;
+			if (util_parse_ranges(optarg, &pd.ranges, &max)) {
```

Comment:
Seems like `max` is always { 0..2^64-1 }. So, what is the point in passing it to *util_parse_ranges* as an argument?